cmd/restic/cmd_backup.go

```diff
@@ -520,6 +525,19 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 	}
 
 	var targetFS fs.FS = fs.Local{}
+	if runtime.GOOS == "windows" && opts.UseWindowsVSS {
+
```

Comment:
Nitpicking, but please remove empty line :)

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/ea769e845ff18c558c098f265b798918e2a4a28d